### ISSUE TITLE
Ontology hover style cause unnecessary redraws

### ISSUE THREAD

**Issue by mccalluc** [parklab/refinery-platform#1404]
-    Specific code commit: 30c8f75
-    Version of the web browser and OS: FF 42.0.1 / Chrome 52.0.2743.116
-    Environment where the error occured: test.stemcellcommons.org:
### Steps to reproduce
1. Do not log in
2. Click "Explore" from the home page
3. Hover over various ontology terms
### Observed behavior
- The "hovered" class adds this CSS:

``` css
.refinery-dashboard .refinery-panel ul.results>li.hovered {
    padding-left: .4rem;
    padding-right: 2.6rem;
    box-shadow: inset .2rem 0 0 0 #000;
}
```

The change in padding causes the content to shift around, sometimes causing the line wrapping to change, and this can cause all the entries below to move.

<img width="431" alt="screen shot 2016-09-15 at 9 38 39 am" src="https://cloud.githubusercontent.com/assets/730388/18552259/aa0613dc-7b29-11e6-902c-94ea33e3b9e6.png">
### Expected behavior
- CSS only adds a shadow on the left, without changing the positioning?

**Comment by mccalluc** [comment]
(We talked, and there should have been compensatory padding on the right so there would not be vertical shifts, but that got lost somewhere along the way.)

**Comment by flekschas** [comment]
Fixed in c2c6064206df51b4b6440a380f53b675f6045a59

**Comment by mccalluc** [comment]
Just confirmed on beta that this is no longer a problem.
